_data/accommodation.yml

```diff
@@ -10,8 +10,8 @@ websites:
   country: DE
   doc: https://www.aohostels.com/en/services/frequent-guest-advantage/
   keywords:
-  - bitpay
-  - bip70
+    - bitpay
```

Comment:
the spacing was proper on these. I used Ruby to reorder them and spit back out updated YAML files so I know that the spacing was right before. Why did you feel it needing changing and not everywhere?

Reply:
I forgot to disable my "Prettier" plugin during the first commit so it automatically got changed when I saved the file. The tabs are reverted in the "Revert tab from first commit in accommodation.yml" commit.

Reply:
Thank you for your contribution. Please make another revert though as this page would say my comment was out of date if it were not still having that additional spacing in a few spots on this YML file.